tests/unit/utils/guildRole.test.ts

```diff
@@ -129,3 +132,71 @@ describe("removeGuildRole", () => {
     expect(result).toEqual(response.INTERNAL_SERVER_ERROR);
   });
 });
+describe("getGuildRoles", () => {
+  it("should return undefined if status is not ok", async () => {
+    jest
+      .spyOn(global, "fetch")
+      .mockImplementationOnce(async () =>
+        Promise.resolve(new JSONResponse({}, { status: 500 }))
+      );
+
+    const response = await getGuildRoles(guildEnv);
+    expect(response).toBeUndefined();
+  });
+
+  it("should return array of objects containing role_id and role_name", async () => {
+    jest
+      .spyOn(global, "fetch")
+      .mockImplementationOnce(async () =>
+        Promise.resolve(new JSONResponse(guildDetailsMock))
+      );
+    const roles = await getGuildRoles(guildEnv);
+    const expectedRoles = [
+      {
+        id: "1234567889",
+        name: "@everyone",
+      },
+      {
+        id: "12344567",
+        name: "bot one",
+      },
+    ];
+    expect(roles).toEqual(expectedRoles);
+  });
+});
+
+describe("getGuildRolesByName", () => {
+  it("should throw role fetch failed message if status is not ok", async () => {
+    jest
+      .spyOn(global, "fetch")
+      .mockImplementationOnce(async () =>
+        Promise.resolve(new JSONResponse({}, { status: 500 }))
+      );
+    const response = await getGuildRoleByName("@everyone", guildEnv);
+    expect(response).toBeUndefined();
+  });
```

Comment:
I don't see an error being thrown if response is not ok. How and where is this scenario in `utils/guildRole`?

Reply:
The message is misleading here. 
Earlier I was throwing errors and hence I wrote the test name like that.
But later on, I changed the code not to throw instead return undefined but forgot to rename the test.
I will fix it in the next commit.